rwengine/src/ai/CharacterController.hpp

```diff
@@ -182,6 +200,23 @@ struct GoTo : public CharacterController::Activity {
     }
 };
 
+struct DriveTo : public CharacterController::Activity {
+    DECL_ACTIVITY(DriveTo)
+
+    AIGraphNode* targetNode;
+    bool rampant; // Drive fast, ignore traffic rules @todo
+
+    DriveTo(AIGraphNode* targetNode, bool _rampant = false)
```

Comment:
targetNode should by default initialized to `nullptr`

Reply:
Maybe it's the culprit of crash.

Reply:
Doesn't hurt to try.